resources/crm-local-develop-environment/localstack-init/setup-aws-services.sh

```
@@ -0,0 +1,315 @@
+    --event-pattern '{"source":["local.email"],"detail-type":["Email Sent","Email Delivered","Email Bounced"]}'
+
+echo "✅ EventBridge rules created"
+
+# Create targets for the rules (connect to SQS queues)
+echo "🎯 Setting up EventBridge targets..."
+
+# User events -> Notification queue
+awslocal events put-targets \
+    --event-bus-name user-event-bus \
+    --rule user-activity-rule \
+    --targets "Id"="1","Arn"="$NOTIFICATION_SQS_ARN"
```

Shellcheck (0.10.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)